src/main/java/uk/gov/hmcts/reform/sscs/controller/CreateCaseController.java

```diff
@@ -129,10 +132,26 @@ private SscsCaseData createSscsCase(String email, String mobile, String hearingT
                     )
                     .build();
             sscsCaseData.getAppeal().setHearingType(hearingType);
+            sscsCaseData.getAppeal().getAppellant().getIdentity().setNino(getRandomNino());
+            sscsCaseData.getAppeal().getMrnDetails().setMrnDate(getRandomMrnDate());
         } catch (IOException e) {
             throw new CreateCaseException(e);
         }
 
         return sscsCaseData;
     }
+
+    @SuppressWarnings("squid:S2245")
+    public String getRandomNino() {
+        return RandomStringUtils.random(9, true, true).toUpperCase();
```

Comment:
Very minor thing but this method already exists in a helper class here: SubmitHelper > getRandomNino()

Reply:
It's in the e2e source folder. It cannot be used here.